### PR TITLE
Settings to edit urls

### PR DESCRIPTION
**Pull request by ichbinbenni**
Adds the possibility to change the target urls of the editors. This will allow educational institutions to run an instance of the website in the local area network. This is useful if the institution has no internet connection to connect to the official web sites. 

Updated to Swift. 4.2

### CODE REVIEW

**Comment by joernalraun** [comment]
Great idea!
However, I am just curious if you could set the initial link to: https://lab.open-roberta.org/#loadSystem&&calliope2017​ so it would jump directly to the Calliope mini editor.

**Comment by spqrPh** [comment]
That would be amazing, we are currently setting this up and tried to get this to work by configuring our Open Roberta installation, simply using this link would solve this for us.

**Comment by spqrPh** [comment]
ok that was fast, thanks

**Comment by ichbinbenni** [comment]
@spqrPh It is a really strange bug!
1. Open the app for the first time. Go to "Start coding" -> Roberta -> Empty screen because swift say the url is not valid.
2. Go to the settings change the url to 2016 in the ending.
3. Repeat step one without closing the App: Everything is fine
4. Change the url back to 2017.
5. Repeat step 1 again without closing the app: This time it's fine too.

WTF! Any idea?

File: Calliope/URLManager.swift [hunk 1]
```diff
@@ -12,7 +12,7 @@ class URLManager {
     
     private static let calliopeDefault = "https://miniedit.calliope.cc"
     private static let makeCodeDefault = "https://makecode.calliope.cc/"
-    private static let robertaDefault = "https://lab.open-roberta.org/#loadSystem&&calliope2017​"
+    private static let robertaDefault = "https://lab.open-roberta.org/#loadSystem&&calliope2017"
```
Comment (spqrPh) [inline Calliope/URLManager.swift]: ??? works now for me though

Comment (ichbinbenni) [reply Calliope/URLManager.swift]: I don't know. Im confused. 🤣

Comment (joernalraun) [reply Calliope/URLManager.swift]: > ok that was fast, thanks
> 
> i just noticed that the link does not work though, i think the year should be 2016 at least then it works.
>
> so https://lab.open-roberta.org/#loadSystem&&calliope2016 @joernalraun can you confirm this?
> 
> (so i guess 2016 is the v.1.3 revision and the other one is the 1.0 one?)

2017 is the latest one – and the one we all want. ;)

Comment (joernalraun) [reply Calliope/URLManager.swift]: Seems to run well. I'll merge the pull request right now and check everything for the coming update. @ichbinbenni please send me a mail if you like – as I have no time to take care of this right now, but have some ideas for some rather quick enhancements you might find useful when using the app.

**Pull request merged**